integration-tests/test-addons/addon-without-jandex/src/main/java/org/vaadin/nojandex/HelloWorld.java

```diff
@@ -0,0 +1,18 @@
+package org.vaadin.nojandex;
+
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.component.littemplate.LitTemplate;
+
+@Tag("hello-world")
+@NpmPackage(value = "@axa-ch/input-text", version = "4.3.11")
+@JsModule("./src/hello-world.ts")
+public class HelloWorld extends LitTemplate {
```

Comment:
As there is no usage of `@Id` this could just be a `Component` to make it lighter as then we don't parse the template which is not needed as no LitTemplate features are used.

Reply:
done. Thanks for explanation